lib/probability/version.rb

```diff
@@ -1,3 +1,3 @@
 module Probability
-  VERSION = "1.0.0"
+  VERSION = "2.0.0"
```

Comment:
:fire: Don't change the version number in a pull request! Ever!
It's not up to you to decide when and how the next version gets released. Please remove all changes regarding version from the pull request.